docs/pages/machine-id/troubleshooting.mdx

```diff
@@ -292,3 +292,46 @@ flag:
 $ tctl bots rm example
 $ tctl bots add example --roles=foo,bar,machine-id-db
 ```
+
+## Kubernetes connections are failing with `Unable to connect to the server: x509: certificate signed by unknown authority`
+
+### Symptoms
+
+A self-hosted cluster is connecting Machine ID to Kubernetes clusters with the following errors.  This can happen for non-TLS configured 
+Teleport clusters.
+
+```bash
+E0322 22:53:31.653051    1699 memcache.go:265] couldn't get current server API group list: Get "https://teleport.example.com:443/api?timeout=32s": x509: certificate signed by unknown authority
+```
+
+To confirm the TLS routing mode check the `proxy.tls_routing_enabled` from this command with your proxy address. 
+If the value is `false` then this is a non-TLS routing configuration.
+```bash
+curl https://teleport.example.com:443/webapi/ping | jq
```

Comment:
1. check the `proxy.tls_routing_enabled` _what_? Key? Check the value of that key/config?
2. Move the explanation of the output to after the command, and use a `:` before.
3. Keep newlines around code blocks. If/when (most likely when) we move to tooling with standard MDX rendering, adherence to Markdown standards will matter.

````suggestion
To confirm the TLS routing mode check the value of the `proxy.tls_routing_enabled` key with this command, substituting your proxy address:

```bash
curl https://teleport.example.com:443/webapi/ping | jq
```

If the value is `false` then this is a non-TLS routing configuration.
````